package.json

```diff
@@ -2,21 +2,22 @@
   "name": "app",
   "private": "true",
   "dependencies": {
-    "@hotwired/stimulus": "^3.2.2",
-    "@hotwired/turbo-rails": "^8.0.4",
-    "autoprefixer": "^10.4.16",
-    "chart.js": "^4.4.1",
-    "class-variance-authority": "^0.7.0",
-    "clsx": "^2.0.0",
-    "esbuild": "^0.19.4",
-    "lottie-web": "^5.12.2",
-    "phlex_ui": "^0.1.2",
-    "postcss": "^8.4.31",
-    "tailwindcss": "^3.3.3",
-    "tailwindcss-animate": "^1.0.7"
+    "@hotwired/stimulus": "3.2.2",
+    "@hotwired/turbo-rails": "8.0.5",
+    "autoprefixer": "10.4.19",
+    "chart.js": "4.4.3",
+    "class-variance-authority": "0.7.0",
+    "clsx": "2.1.1",
+    "esbuild": "0.23.0",
+    "lottie-web": "5.12.2",
+    "phlex_ui": "0.1.5",
+    "postcss": "8.4.40",
+    "tailwindcss": "3.4.7",
+    "tailwindcss-animate": "1.0.7"
   },
   "scripts": {
-    "build": "esbuild app/javascript/*.* --bundle --sourcemap --outdir=app/assets/builds --public-path=/assets",
+    "build": "esbuild app/javascript/*.* --bundle --sourcemap --format=esm --outdir=app/assets/builds --public-path=/assets",
     "build:css": "tailwindcss -i ./app/assets/stylesheets/application.tailwind.css -o ./app/assets/builds/application.css --minify"
-  }
-}
+  },
+  "packageManager": "yarn@1.22.19"
```

Comment:
this might have something to do with the yarn release files. Look into removing this.

Reply:
This was also added by the Fly CLI. It seems to be a good practice to specify the Yarn version here to ensure that the specific Yarn version is always used.

<img width="1069" alt="Screenshot 2024-07-29 at 11 51 37 AM" src="https://github.com/user-attachments/assets/990af696-a5bd-4ff3-be4e-be8e3ab0f209">